hivemind/p2p/p2p_daemon.py

```diff
@@ -95,6 +95,7 @@ async def create(
         use_relay: bool = True,
         use_relay_hop: bool = False,
         use_relay_discovery: bool = False,
+        persistent_conn_max_msg_size: int = 2 << 22,
```

Comment:
1. The default `2 << 22` does not match the Go default `1 << 22` (seems like a typo).

2. Let's write it as `4 * 1024 ** 2`, so it is more clear that this is 4 MiB.

3. Having the default in 3 places is misleading: if a user changes one of them that they accidentally find, the behavior won't change as expected. Let's create the constant `DEFAULT_MAX_MSG_SIZE` in the most inner module (`control.py`), then set all defaults to this constant.